org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/BuildDate.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *    Debrief - the Open Source Maritime Analysis Application
+ *    http://debrief.info
+ *
+ *    (C) 2000-2016, Deep Blue C Technology Ltd
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the Eclipse Public License v1.0
+ *    (http://www.eclipse.org/legal/epl-v10.html)
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ */
+package org.mwc.debrief.lite;
+
+/**
+ * @author Ayesha
+ *
+ */
+public class BuildDate
+{
+  public static final String BUILD_DATE="12 Jun 2019 17:30";

```

Comment:
@ayeshare - I think you're committing the modified file. This one hasn't got the `$DATE_HERE$` marker.

Hmm, I think we're going to keep on getting this issue.  I've got an alternate suggestion:
* we introduce a file titled "BuildDateMaster.tzt" - it contains the current contents of BuildDate.java
* in the build, we make a copy of the file, and rename it to BuildData.java
* we make the change to this file
* we can then mark BuildDate.java as `ignore` in Git - so it won't get flagged up as a changed document after each release.